.github/dependabot.yaml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
```

Comment:
```suggestion
    directory: ".github/workflows"
```
Shouldn't this be limited to the `.github/workflows` sub-directory?

Reply:
No, the directory is correct.